src/topp/CometAdapter.cpp

```diff
@@ -751,6 +752,9 @@ class TOPPCometAdapter :
     // if "reindex" parameter is set to true will perform reindexing
     if (auto ret = reindex_(protein_identifications, peptide_identifications); ret != EXECUTION_OK) return ret;
 
+	// Parse IM information
+ 	SpectrumMetaDataLookup::addMissingIMToPeptideIDs(peptide_identifications, inputfile_name, true);
```

Comment:
what if the data has no IM?
CometAdapter crashes?

Reply:
Works with the current commit